### DISCUSSION TITLE
Persistence module - 'get' function not working

### DISCUSSION THREAD

**Discussion by azm08** [anvilistas/anvil-extras#522] [Q&A] [answered]
Hi,
I've created a quick clone to test the peristence module.
I am not able to make the ‘get’ function work which returns me the following error:
anvil.tables.TableError: Column ‘title’ can only be searched with a string (not a simpleObject) in table ‘book’
But if I use the ‘search’ feature, it works fine.
Can't seem to understand why it doesn't work.
Thanks for your help
A quick clone of the app:
https://anvil.works/build#clone:SUFWHYTXNEP6XU5E=6COQXS3VUWW4O3SJSVR37PFN

**Comment by meatballs** [comment, marked as answer]
Looks like you've found a bug. I've created an issue for it: #523 

For the moment, change line 11 of your server module to:
```py
return app_tables.book.get(**title)
```

until we get it fixed.
